Added create trace pipeline method (Breaking Change: removes implementation of "SpanBatcher") (#56)
GoogleCloudPlatform/opentelemetry-operations-go: exporter/trace/README.md

import texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
```

Once you import the trace exporter package, create and install a new export pipeline, and then you can start tracing. If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instruction in [Authentication](#Authentication).

```go
// Create exporter and trace provider pipeline, and register provider.
projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
_, flush, err := texporter.InstallNewPipeline(
        []texporter.Option {
                texporter.WithProjectID(projectID),
                // other optional exporter options
        },

>         // For the demonstration, use sdktrace.AlwaysSample sampler to sample all traces.

Reviewer: ```suggestion
        // This example code uses sdktrace.AlwaysSample sampler to sample all traces.
```
